### PR TITLE
Added support for alternative ssh ports

### PR DESCRIPTION
**Pull request by nubz**
I had a need to use a port number other than 22 for my deployment, I have edited this to support the use of STAGING_SSH_PORT, PROD_SSH_PORT etc in WP-config.php

### CODE REVIEW

File: lib/command.php [hunk 1]
```diff
@@ -22,6 +22,8 @@ public function push( $args = array() ) {
 			WP_CLI::error( "$env environment is locked, you cannot push to it" );
 			return;
 		}
+		
+		$ssh_port = ($ssh_port) ? $ssh_port : 22;
```
Comment (demental) [inline lib/command.php]: Maybe this could be moved to _prepare_and_extract() for DRY.

**Comment by demental** [comment]
Nice feature, apart from the small enhancement requested in my last comment !

File: lib/command.php (hunk 1 above)
Comment (nubz) [inline lib/command.php]: for sure

**Comment by nubz** [comment]
the quotes are required for rsync ssh but not for the constant e.g. define('STAGING_SSH_PORT', 223); works just fine - perhaps some typecasting would help

EDIT - I see its my use of the escapeshellarg method that's adding the extra quotes, I get no complaint from Ubuntu with the quotes in but can't say how it works elsewhere - how about I don't escapeshellarg it and just intval() instead?

**Pull request merged**